examples/connext_dds/content_filtered_topic/c++11/cft_subscriber.cxx

```diff
@@ -72,95 +70,101 @@ void subscriber_main(int domain_id, int sample_count, bool is_cft)
     // The Content Filter Expresion has two parameters:
     // - %0 -- x must be greater or equal than %0.
     // - %1 -- x must be less or equal than %1.
-    ContentFilteredTopic<cft> cft_topic = dds::core::null;
-    DataReader<cft> reader = dds::core::null;
+    dds::topic::ContentFilteredTopic<cft> cft_topic = dds::core::null;
+    dds::sub::DataReader<cft> reader = dds::core::null;
+    dds::sub::Subscriber subscriber(participant);
 
     if (is_cft) {
         std::cout << "Using ContentFiltered Topic" << std::endl;
-        cft_topic = ContentFilteredTopic<cft>(
+        cft_topic = dds::topic::ContentFilteredTopic<cft>(
                 topic,
                 "ContentFilteredTopic",
-                Filter("x >= %0 AND x <= %1", parameters));
-        reader =
-                DataReader<cft>(Subscriber(participant), cft_topic, reader_qos);
+                dds::topic::Filter("x >= %0 AND x <= %1", parameters));
```

Comment:
I don't think declaring the parameters as a separate variable serves any purpose.

```suggestion
                dds::topic::Filter("x >= %0 AND x <= %1", {1, 4}));
```